src/lib/components/ui/ILSFooter/ILSFooter.js

```diff
@@ -0,0 +1,71 @@
+import { getStaticPageByName } from "@config/uiConfig";
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import Overridable from "react-overridable";
+import { Link } from "react-router-dom";
+import { Container, Grid, Header, List } from "semantic-ui-react";
+
+const OverridableFooter = props => {
```

Comment:
not sure I understand the overridable pattern, you are basically defining 2 overridable templates that are the same.
It should be:
```
class Footer extends Component {
  render() {
    return <Overridable id="Footer.container" {...this.props}>...</>;
  }
}

export default Overridable.component("Footer", Footer);
```

this is valid for all the components here

```